src/transactions/transaction_builder/transaction_builder.ts

```diff
@@ -35,15 +35,15 @@ import {
   TransactionAuthenticatorEd25519,
   TransactionAuthenticatorFeePayer,
   TransactionAuthenticatorMultiAgent,
-  SingleSenderTransactionAuthenticator,
+  TransactionAuthenticatorSingleSender,
```

Comment:
This change makes sense for consistency, but why was it made in the first place without the type first?

Reply:
I'm not sure- I think Maayan and I were just trying to get it working and it slipped in like this.

Tbh I don't remember when it even became this, I was confused when I saw it

Reply:
Not that they're both different ways, but that if you were to choose a name not thinking of either one.  Which one would you choose:

1. TypeTransactionAuthenticator
2. TransactionAuthenticatorType

That should be the one you choose (since we're just breaking anything over better UX)

Reply:
In general I prefer the more specific descriptor first, because when you're typing auto complete works faster that way haha

And visually it's easier to see what it is that way